day1/exercise1/solution.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import "fmt"
+
+type matrix struct {
+	row, col int
```

Comment:
always create them as separate properties although row and col are int type.

Reply:
type matrix struct {
	row int
        col int
        val [] [] int
       }

Reply:
cool